docs/en/docs/templates.md

```diff
@@ -13,9 +13,9 @@ from lilya.templating import Jinja2Template
 
 ### Parameters
 
-- `directory`: A string, [os.Pathlike][pathlike], or a list of strings or [os.Pathlike][pathlike] indicating a directory path.
-- `env`: Any different `jinja2.Environment` instance *(Optional)*.
-- `**options`: Additional keyword arguments to pass to the Jinja2 environment.
+-   `directory`: A string, [os.Pathlike][pathlike], or a list of strings or [os.Pathlike][pathlike] indicating a directory path.
+-   `env`: Any different `jinja2.Environment` instance _(Optional)_.
```

Comment:
`env`: Any different `jinja2.Environment` instance *(Optional)* please, this will make it italic in markdown.

Reply:
fixed